ownership-chain/runtime/src/tests/spec_version.rs

```diff
@@ -0,0 +1,108 @@
+use crate::VERSION;
```

Comment:
Why did you implement this inside `tests` folder? I would move this file to `runtime` module instead

Reply:
runtime tests are consolidated in the `tests` folder

Reply:
it includes logic not only tests, isn't it?

Reply:
you mean `version_to_int` function? if yes, see my comment below